### ISSUE TITLE
wrong textures for shared chest in mcl/mcla

### ISSUE THREAD

**Issue by PiTTl** [SmallJoker/simple_protection#37]
hi,
![trapped_chest](https://github.com/user-attachments/assets/025abd05-97e0-4ed2-bb58-9e183572ba76)

```
"mcl_chests_chest_trapped_top.png", 
"mcl_chests_chest_trapped_bottom.png", 
"mcl_chests_chest_trapped_right.png", 
"mcl_chests_chest_trapped_left.png", 
"mcl_chests_chest_trapped_back.png", 
"mcl_chests_chest_trapped_front.png", 
```
doesn't exist in VoxeLibre/MCL2 / Mineclonia.
the current texture for the small chest is
"mcl_chests_trapped.png"
and looks like this:
![mcl_chests_trapped](https://github.com/user-attachments/assets/99630034-086f-490e-bb9d-4d3b21c99a83)

**Comment by SmallJoker** [comment]
Fixed in 286d3c7

**Comment by PiTTl** [comment]
Thank you  :-)
